test/integration-tests/extension.test.ts

```diff
@@ -67,11 +67,7 @@ suite("Extension Test Suite", () => {
 
     suite("Workspace", () => {
         /** Verify tasks.json is being loaded */
-        test("Tasks.json", async () => {
-            // Skip if running CI as it takes too long
-            if (process.env.CI) {
-                return;
-            }
+        test("Tasks.json @slow", async () => {
```

Comment:
is this one slow? not running the task

Reply:
I was just going off the comment: `// Skip if running CI as it takes too long`

Reply:
hmm, assumed that was new but looks like that skip was added 14 months ago. That doesn't make sense to me, should in theory be quick, maybe we should change how globalWorkspaceContentPromise resolves. In theory once extension is activated this step should be almost instantaneous

Reply:
You're right, this is a fast test, updated.